src/main/java/me/totalfreedom/totalfreedommod/command/Command_whohas.java

```diff
@@ -32,7 +32,7 @@ public boolean run(CommandSender sender, Player playerSender, Command cmd, Strin
 
         if (material == null)
         {
-            msg("Invalid item: " + materialName, ChatColor.RED);
+            msg("Invalid item: " + materialName + ".", ChatColor.RED);
```

Comment:
Think this could get confusing having a variable next to a full stop?

Reply:
fixed